database/seeds.js

```diff
@@ -1,32 +1,58 @@
 const faker = require('faker');
 const db = require('../database');
 
-// // generate 100 fake user
-for (let i = 0; i < 100; i++) {
-  const userName = faker.internet.userName();
-  const userEmail = faker.internet.email();
-  const userPassword = faker.internet.password();
-  db.query(`INSERT INTO users (name,email,password) VALUES ("${userName}","${userEmail}","${userPassword}");`);
+const menu = [
+  [1, 'bread', 4.5, ''],
+  [2, 'soup', 1.5, 'cheicken, pease, carrot, onion'],
+  [3, 'california roll', 12, 'Masago, crabmeat, cucumber and avocado'],
+  [4, 'B.L.T. Sandwich', 9.75, 'Bacon, lettuce and tomatoes'],
+  [5, 'Veggie Burger', 8, 'Served on a toasted bun with coleslaw and pickles'],
+];
+
+// function to get random elements from menu array
+function getRandomMenuItems() {
+  let result = {};
```

Comment:
Are you using an object instead of an array to avoid duplicates? If so, you can use `Set` which is cleaner (you don't need to set values you aren't going to use).